dex-it/src/test/scala/com/wavesplatform/it/matcher/api/http/cancel/CancelOrdersByAddressAndIdsSpec.scala

```diff
@@ -108,43 +93,16 @@ class CancelOrdersByAddressAndIdsSpec extends MatcherSuiteBase with ApiKeyHeader
 
     "should return an error when address is not a correct base58 string" in {
       validateMatcherError(
-        dex1.rawApi.cancelOrdersByIdsWithKeyOrSignature("null", placeAndGetIds(3)),
+        dex1.rawApi.cancelOrdersByIdsWithKey(placeAndGetIds(3).map(_.toString), "null"),
         StatusCode.BadRequest,
         InvalidAddress.code,
         "Provided address is not correct, reason: Unable to decode base58: requirement failed: Wrong char 'l' in Base58 string 'null'"
       )
     }
 
-    "should return order not found if address path parameter and publicKey header are from different accounts" in {
```

Comment:
Why do we remove this these tests (below too)? They look correct

Reply:
it comes after merging master.
this tests are in another suite and it needn't here (look at api-method they test)